spec/models/product_importer_spec.rb

```diff
@@ -532,20 +532,15 @@
 
   describe "when more than one product of same name already exists with multiple variants each" do
     let(:csv_data) {
-      CSV.generate do |csv|
-        csv << ["name", "producer", "category", "description", "on_hand", "price", "units",
-                "unit_type", "display_name", "shipping_category"]
-        csv << ["Oats", enterprise.name, "Cereal", "", "50", "3.50", "500",
-                "g", "Rolled Oats", shipping_category.name]   # Update
-        csv << ["Oats", enterprise.name, "Cereal", "", "80", "3.75", "500",
-                "g", "Flaked Oats", shipping_category.name]   # Update
-        csv << ["Oats", enterprise.name, "Cereal", "", "60", "5.50", "500",
-                "g", "Magic Oats", shipping_category.name]    # Add
-        csv << ["Oats", enterprise.name, "Cereal", "", "70", "8.50", "500",
-                "g", "French Oats", shipping_category.name]   # Add
-        csv << ["Oats", enterprise.name, "Cereal", "", "70", "8.50", "500",
-                "g", "Scottish Oats", shipping_category.name] # Add
-      end
+      csv_doc = <<~CSV
+        name,producer,category,description,on_hand,price,units,unit_type,display_name,shipping_category
+        Oats,#{enterprise.name},Cereal,,50,3.50,500,g,Rolled Oats,#{shipping_category.name}
+        Oats,#{enterprise.name},Cereal,,80,3.75,500,g,Flaked Oats,#{shipping_category.name}
+        Oats,#{enterprise.name},Cereal,,60,5.50,500,g,Magic Oats,#{shipping_category.name}
+        Oats,#{enterprise.name},Cereal,,70,8.50,500,g,French Oats,#{shipping_category.name}
+        Oats,#{enterprise.name},Cereal,,70,8.50,500,g,Scottish Oats,#{shipping_category.name}
+      CSV
+      CSV.generate { |csv| csv_doc.each_line { |line| csv << line.strip.split(",") } }
```

Comment:
What's the difference between `csv_doc` and `csv_data`? Just wondering if we need this line. It doesn't hurt though.

Reply:
For some strange reason I thought I needed to generate it. It works without it though so I removed it.